### PR TITLE
Add historic

### PR DESCRIPTION
**Pull request by 79Inaf**
Added table historic + it's generated in SQL with the historic of the player + No back button + trumpet sound

### CODE REVIEW

File: src/Controller/ChapterController.php [hunk 1]
```diff
@@ -44,8 +45,11 @@ public function adminIndex(): string
         return $this->twig->render('Chapter/admin_index.html.twig', ['chapters' => $chapters]);
     }
 
-    public function showWithAction(int $id): string
+    public function showWithAction(int $id, int | null $action = null): string
     {
+        $historicManager = new HistoricManager();
+        $historicManager->historicInsert($action);
```
Comment (aldaitzwild) [inline src/Controller/ChapterController.php]: You should'nt call this if action is null

File: src/Controller/HistoricController [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//         if ($_SERVER['REQUEST_METHOD'] === 'GET') {
+//             $historic = $_GET;
+//             $historicManager = new HistoricManager();
+//             $id = $historicManager->historicInsert($historic);
+
+//             return null;
+//         }
+
+//     }
+// }
```
Comment (aldaitzwild) [inline src/Controller/HistoricController]: A full file of dead code ! Congrats

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Model/HistoricManager.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace App\Model;
+
+use PDO;
+
+class HistoricManager extends AbstractManager
+{
+    public const TABLE = 'historic';
+
+    public function historicInsert(int | null $actionId): void
```
Comment (aldaitzwild) [inline src/Model/HistoricManager.php]: No $action should'nt be allowed to be null

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflict With Dev

File: af_database.sql [hunk 4]
```diff
@@ -93,10 +93,30 @@ VALUES
 ("Essayez encore !", 15, NULL),
 ("Félicitations !", 16, NULL);
 
+SELECT * FROM historic;
+
+TRUNCATE historic;
```
Comment (aldaitzwild) [inline af_database.sql]: It should be TRUNCATE TABLE

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: af_database.sql (hunk 4 above)
Comment (79Inaf) [reply af_database.sql]: I fixed the problem on the file af_database.sql

**Comment by charlietoussaint** [comment]
We decided to close this PR for a new one with CSS, advised fixed are implemented

**Pull request closed without merging**